documentation/src/docs/security/pravega-security-configurations.md

```diff
@@ -47,6 +47,16 @@ their Transport Layer Security (TLS) and auth (short for authentication and auth
    |Valid values:|{`true`, `false`} |
    |Old name:|`controller.auth.tlsEnabled` (deprecated) |
     
+* __<ins>controller.security.tls.protocolVersion</ins>__
+
+   |Property| Value | 
+   |---:|:----|
+   |Description: | Configurable versions for TLS Protocol |
+   |Type:|String|
+   |Default:| `TLSv1.2,TLSv1.3`|
+   |Valid values:|{`TLSv1.2`, `TLSv1.3`} |
```

Comment:
Can you have a valid value containing both? If so, we should include that too.

Reply:
Fixed. Included all valid values.